### DISCUSSION TITLE
argo-server api ratelimit

### DISCUSSION THREAD

**Discussion by karthik-desynova** [argoproj/argo-workflows#7879] [Q&A]
if im using submit endpoint **$host/api/v1/workflows/{namespace}/submit.**
are there are any rate limit to API itself, i searched through documentation but i could not  find anything.

**Comment by terrytangyuan** [comment]
I don't think we have any limit on Argo Server. Also this might be helpful https://argoproj.github.io/argo-workflows/running-at-massive-scale/
